### PR TITLE
including gritzko and joaomantovani contributing

### PR DESCRIPTION
**Pull request by gritzkoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,11 +53,17 @@ Dextra is a Software development company which is really proud of the great work
 
  - [Phoenix Burst Wallet](https://github.com/burst-apps-team/phoenix) - Cross-platform crypto coin wallet for the Burstcoin blockchain. Co-Author/Maintainer: [Oliver Hager](https://github.com/ohager)
  - [@burstjs](https://burst-apps-team.github.io/phoenix) - Reference Library for Burstcoin blockchain platform. Co-Author/Maintainer: [Oliver Hager](https://github.com/ohager)
+ 
+ - [nodejs-health-checker](https://github.com/gritzkoo/nodejs-health-checker) - A package that allows you to track your application healthy providing two ways of checking, with and without integrations, to standardize implementations cross microservices and for applications that are running in k8s, provide liveness and readiness approach. Authors: [Gritzko D. Kleiner](https://github.com/gritzkoo) and [João Pedro Mantovani](https://github.com/joaomantovani)
```
Comment (erickzanardo) [inline README.md:57]: Suggestion: instead of `your application healthy` I would go with `your application health`

File: README.md [hunk 2]
```diff
@@ -53,11 +53,17 @@ Dextra is a Software development company which is really proud of the great work
...
 
 <h1>.NET <img style="margin: 4px 0 0 4px" height="32" src="media/dotnet.svg" alt=".NET Logo"/></h1>
 
  - [DexQuiz](https://github.com/dextra/dexquiz) - A simple quiz with different tracks, to be used in events to engage people with the company and also help with gifts and awards. Authors: [Gustavo Bigardi](https://github.com/gustavobigardi), [Gabriel Cordeiro](https://github.com/Gabriel-Cordeiro), [Bruno Habbermann](https://github.com/bhabermann) and [Alexandre Bonfitto](https://github.com/alexandrehtrb)
 
+<h1>Golang <img style="margin: 4px 0 0 4px" height="32" src="media/go.svg" alt=".NET Logo"/></h1>
+
+ - [golang-health-checker](https://github.com/gritzkoo/golang-health-checker) - A package that allows you to track your application healthy providing two ways of checking, with and without integrations, to standardize implementations cross microservices and for applications that are running in k8s, provide liveness and readiness approach. Author: [Gritzko D. Kleiner](https://github.com/gritzkoo)
```
Comment (erickzanardo) [inline README.md:65]: Same suggestion here.

File: README.md [hunk 3]
```diff
@@ -53,11 +53,17 @@ Dextra is a Software development company which is really proud of the great work
...
+ 
+ - [nodejs-health-checker](https://github.com/gritzkoo/nodejs-health-checker) - A package that allows you to track your application healthy providing two ways of checking, with and without integrations, to standardize implementations cross microservices and for applications that are running in k8s, provide liveness and readiness approach. Authors: [Gritzko D. Kleiner](https://github.com/gritzkoo) and [João Pedro Mantovani](https://github.com/joaomantovani)
 
 <h1>.NET <img style="margin: 4px 0 0 4px" height="32" src="media/dotnet.svg" alt=".NET Logo"/></h1>
 
  - [DexQuiz](https://github.com/dextra/dexquiz) - A simple quiz with different tracks, to be used in events to engage people with the company and also help with gifts and awards. Authors: [Gustavo Bigardi](https://github.com/gustavobigardi), [Gabriel Cordeiro](https://github.com/Gabriel-Cordeiro), [Bruno Habbermann](https://github.com/bhabermann) and [Alexandre Bonfitto](https://github.com/alexandrehtrb)
 
+<h1>Golang <img style="margin: 4px 0 0 4px" height="32" src="media/go.svg" alt=".NET Logo"/></h1>
```
Comment (erickzanardo) [inline README.md:63]: Lets keep categories on alphabetical order too

**Review by erickzanardo — CHANGES_REQUESTED** [review]
Just need to fix the new category order to match the alphabetical order. Other than that, LGTM

**Review by erickzanardo — APPROVED** [review]
LGTM

**Review by ohager — APPROVED** [review]
cool

**Pull request merged**